cmd/openshift-tests/e2e.go

```diff
@@ -50,7 +50,7 @@ var staticSuites = []*ginkgo.TestSuite{
 		The disruptive test suite.
 		`),
 		Matches: func(name string) bool {
-			return strings.Contains(name, "[Disruptive]") && strings.Contains(name, "[Feature:EtcdRecovery]")
+			return strings.Contains(name, "[Disruptive]") && strings.Contains(name, "master and worker failure and recover with machine health checks")
```

Comment:
Now that we're getting close, this should be the new set of tests we want to run:

```
return strings.Contains(name, "[Feature:EtcdRecovery]") || strings.Contains(name, "[Feature:NodeRecovery]")
```